data/binding/binding.go

```diff
@@ -44,6 +50,12 @@ func (b *base) AddListener(l DataListener) {
 	b.lock.Lock()
 	defer b.lock.Unlock()
 
+	for _, listen := range b.listeners { // TODO maybe fix? workaround for list not being able to de-register
+		if listen == l {
+			return
+		}
+	}
```

Comment:
Can you elaborate on this issue? What breaks if this is removed?

Reply:
Nothing, it was a hangover from earlier revision, will remove